### DISCUSSION TITLE
Congrats! 🎉

### DISCUSSION THREAD

**Discussion by mateus4k** [adonisjs/core#2475] [General]
Congratulations on 10k stars on Github.

This project is incredible and very useful for the community! I've been using Adonis for a long time on the [Sabiá Platform](https://github.com/ufersa/plataforma-sabia) and I would really like this framework to be more recognized in the NodeJs community!

Thank you very much for all the effort, @thetutlage and @RomainLanz! 🚀

**Comment by sn0rcha** [comment]
Congratulations guys, this project is amazing!

I've been using it for a number of years on many systems. An absolute pleasure to code with.

Thanks for all the effort and support @thetutlage and @RomainLanz

**Comment by maidiries** [comment]
Nice conversation thread! I would like to add my congratulations to those
